scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -66,7 +65,8 @@ class PostgresStorage(
     /**
      * The JDBC data source to obtain database connections.
      */
-    private val dataSource: Lazy<DataSource>
+    private val dataSource: Lazy<DataSource>,
+    private val storageThreads: Int
```

Comment:
Nit: Please at least add a blank like here to make clear that the above docs only apply to line 68, or ideally also add brief docs for line 69.